src/hooks/useUser.ts

```diff
@@ -16,6 +16,10 @@ const useUser = (id: string | undefined): UserReadSerializer | null => {
     if (userId === self.id) {
       return {
         avatar: self.avatar,
+        // user stats are available in users state, not in self state.
+        default_wallet_balance: users[self.id]?.default_wallet_balance ?? undefined,
+        followers_count: users[self.id]?.followers_count ?? undefined,
+        following_count: users[self.id]?.following_count ?? undefined,
```

Comment:
tbh, I don't really like this approach, but since we only have these stats in the `getUser` API, so they aren't available in self (via login, signup).
Anytime we hit `getUser` API for any user, their stats are also fetched and stored.

Reply:
@buckyroberts But if you think we should create a completely separate endpoint for fetching user stats, let me know. We can do that as well